Update documentation for article (#237)
mosdef-hub/foyer: docs/paper_examples.md

### Paper Examples

Contained below are the toy examples from the *Usage Examples* section of the [`foyer` paper](https://arxiv.org/pdf/1812.06779.pdf).
The source code selections are listed below on this page, there are [Jupyter Notebooks](https://github.com/mosdef-hub/foyer/tree/master/docs/examples) where you can try these examples yourself.

> Note that these examples are meant to showcase the abilities of `foyer` through rather simplistic examples.

Reviewer: ```suggestion
Note that these examples are meant to showcase the abilities of `foyer` through simple examples.
```

> If the user would like to examine more in-depth and complex examples, please refer to the [tutorial repository](https://github.com/mosdef-hub/mosdef_tutorials).

Reviewer: ```suggestion
If the user would like to examine more in-depth examples using `foyer` with `mBuild`, please refer to the [tutorial repository](https://github.com/mosdef-hub/mosdef_tutorials).
```

Below is *Listing 6* from the paper, a python script to fill a $$2x2x2 nm$$ box with 100 ethane molecules.
The system is then atomtyped using the OPLS-AA forcefield.
There are two approaches to the same problem detailed below in this listing, the first approach uses the `forcefield_files` function argument from [`mBuild`](https://github.com/mosdef-hub/mbuild) to atomptype the system (using foyer under the hood).

> While the second approach creates a `foyer` `Forcefield` object, which then calls its `apply` function, operating on the `mBuild` `Compound` to return the properly atomtyped structure.

Reviewer: Again with defining that's it a `ParmEd` structure, as distinct from one of ours

```python
import mbuild as mb
from mbuild.examples import Ethane
from foyer.tests.utils import get_fn
from foyer import Forcefield

> ### Approach 1 ###

Reviewer: Same comments as before (I think this is the same code?)

Note that in all instances when using `foyer`, the chemical system of interest is converted into a `ParmEd` `Structure`.
Even the `mBuild` `Compounds`, when calling the `save` routine, are converted into a `ParmEd` `Structure` before `foyer` can atomtype them.
The object returned by `foyer` after the atomtypes have been applied are `ParmEd` `Structures`.
This is subject to change in later iterations of `foyer`, 

```python
import mbuild as mb
from mbuild.examples import Ethane
from foyer.tests.utils import get_fn
from foyer import Forcefield

### Applying a force field while saving from mBuild ###
# Create the chemical topology
ethane_fluid = mb.fill_box(compound=Ethane(), n_compounds=100, box=[2, 2, 2])
# Apply and save the topology
ethane_fluid.save(’ethane-box.top’, forcefield_files=get_fn(’oplsaa_alkane.xml’))
ethane_fluid.save(’ethane-box.gro’)

### Applying a force field directly with foyer ###
# Create the chemical topology
ethane_fluid = mb.fill_box(compound=Ethane(), n_compounds=100, box=[2, 2, 2])
# Load the forcefield
opls_alkane = Forcefield(forcefield_files=get_fn(’oplsaa_alkane.xml’))
# Apply the forcefield to atom-type
ethane_fluid = opls_alkane.apply(ethane_fluid)

> # Save the atom-typedsystem

Reviewer: ```suggestion
# Save the atom-typed system
```
